examples/taint-checking/taint_checking.description

```diff
@@ -1,8 +1,12 @@
-// Ballerina is designed to ensure that programs written in Ballerina are inherently secure. Ballerina programs
-// are resilient to major security vulnerabilities including SQL injection, path manipulation, file manipulation,
-// unauthorized file access, and unvalidated redirect (open redirect).<br/><br/>
+// Ballerina is designed to ensure that programs written in Ballerina are inherently secure.
+// With taint flow analysis (a.k.a taint checking) Ballerina compiler identifies data flows which may lead to
+// untrusted data being passed to security-sensitive areas of the program.
 //
-// A taint analysis mechanism is used to achieve this. As a result of the taint analysis mechanism, the Ballerina compiler
-// identifies untrusted (tainted) data by observing how tainted data propagates through the program. If untrusted data
-// is passed to a security sensitive parameter, a compile error is generated.<br/><br/>
+// By enabling taint checking, Ballerina programs can be made resilient to major security vulnerabilities including
+// SQL injection, path manipulation, file manipulation, unauthorized file access,
+// and unvalidated redirect (open redirect).
 // For more information, see [How to Write Secure Ballerina Programs](https://ballerina.io/swan-lake/learn/how-to-write-secure-ballerina-code/).
+//
+// Taint analysis is disabled by default and it is enabled by providing the `--taint-check` option
```

Comment:
```suggestion
// Taint analysis is disabled by default and it can be enabled by providing the `--taint-check` option
```

Reply:
is using `is` doesn't convey the message here? Or is it a grammar issue?

Reply:
We can't use "is" with both enabled and disabled, right? "is" is what the current state is whereas it can be changed by providing the option?

@praneesha, thoughts please?

Reply:
Will change to "can be".